src/status/devices.nim

```diff
@@ -0,0 +1,25 @@
+import system
+import libstatus/settings
+import libstatus/installations
+import json
+
+proc setDeviceName*(name: string) =
+  discard getSettings()
```

Comment:
Do you really need to do `getSettings` first? I think `getSetting` does it if the is dirty no?